### PR TITLE
Fix lints

### PR DESCRIPTION
**Pull request by andresilva**
This PR is a first pass at cleaning up the `Operations` and `OperationsProxy` contracts. I've fixed all the lints reported by `solium` and all the warnings reported by `solc`. I also fixed a logic bug in the `only_unratified` modifier.

I recommend to review this PR commit-by-commit as I made sure that the commit history is clean and readable.

### CODE REVIEW

File: contracts/Operations.sol [hunk 1]
```diff
@@ -276,38 +276,94 @@ contract Operations is OperationsFace {
...
+		_;
+	}
+
+	modifier only_ratified {
+		require(fork[proposedFork].ratified);
+		_;
+	}
+
+	modifier only_unratified {
+		require(!fork[proposedFork].ratified);
```
Comment (tomusdrw) [inline contracts/Operations.sol:312]: Seems you've actually fixed a bug here :) The original code does the same as `only_ratified`

File: contracts/Operations.sol [hunk 2]
```diff
@@ -276,38 +276,94 @@ contract Operations is OperationsFace {
...
+	}
+
+	modifier only_when_proxy_undecided(bytes32 _txid) {
+		require(proxy[_txid].status[clientOwner[msg.sender]] == Status.Undecided);
+		_;
+	}
+
+
+	modifier when_required(bytes32 _client) {
+		if (client[_client].required)
```
Comment (tomusdrw) [inline contracts/Operations.sol:349]: Weird it doesn't throw :/ But I see that no logic has been changed.

File: contracts/OperationsProxy.sol [hunk 3]
```diff
@@ -61,38 +61,39 @@ contract OperationsProxy {
...
 	}
 
-	function setConfirmer(address _confirmer, uint8 _track) only_owner {
+	function setConfirmer(address _confirmer, uint8 _track) public only_owner {
 		ConfirmerChanged(confirmer[_track], _confirmer, _track);
 		confirmer[_track] = _confirmer;
 	}
 
-	function addRelease(bytes32 _release, uint32 _forkBlock, uint8 _track, uint24 _semver, bool _critical) {
+	function addRelease(bytes32 _release, uint32, uint8 _track, uint24, bool) public {
```
Comment (tomusdrw) [inline contracts/OperationsProxy.sol]: ABI has been changed here, was that intentional?

File: contracts/OperationsProxy.sol [hunk 4]
```diff
@@ -61,38 +61,39 @@ contract OperationsProxy {
...
 		if (relayOrConfirm(_track))
 			AddReleaseRelayed(_track, _release);
 		else
 			trackOfPendingRelease[_release] = _track;
 	}
 
-	function addChecksum(bytes32 _release, bytes32 _platform, bytes32 _checksum) {
+	function addChecksum(bytes32 _release, bytes32 _platform, bytes32) public {
```
Comment (tomusdrw) [inline contracts/OperationsProxy.sol]: Same here, ABI changed.

**Review by tomusdrw — COMMENTED** [review]
Two changes in ABI that I don't know if were intended.

**Comment by andresilva** [comment]
I've re-added the unused parameters in `OperationsProxy` to make sure we don't break the ABI. I'll revisit this in a subsequent PR when cleaning up the `OperationsProxy`.

**Review by tomusdrw — APPROVED** [review]

**Pull request merged**